src/main/resources/errors.yaml

```diff
@@ -903,6 +915,17 @@ request-errors:
       
       Resend the command using columns of the supported data types.
 
+  - scope: SCHEMA
+    code: UNSUPPORTED_INDEXING_FOR_FROZEN_COLUMN
+    title: Index creation on a frozen column is not supported
+    body: |-
+      The command attempted to create an index on a column that is frozen.
+      
+      The table ${keyspace}.${table} defines the columns: ${allColumns}.
+      '${targetColumn}` is frozen on the table.
```

Comment:
nit:   '${targetColumn}` is frozen on the table.  -> to " The table has the frozen column:  ${targetColumn}"